### DISCUSSION TITLE
H->bb contamination in the signal region?

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#44] [Comments_HIG_23_001]
> Did you check the contamination from H->bb in the region above 135 GeV? The window 105-135 GeV seems to be too narrow for a H->bb decay

**Comment by jialin-guo1** [comment]
As we discuss it should be ZH not Hbb. because we have a Z->ll

**Comment by ram1123** [comment]
Yes.  It should be ZH not just H.

**Reply by jialin-guo1** [reply]
I think we checked this before. the contribution is so small. i can make a plot again

**Reply by jialin-guo1** [reply]
in 105-135GeV 

resolved:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/d4515b33-5273-41da-bd7e-99c6a9116067)

merged:
![image](https://github.com/ram1123/2l2Q_limitSettingTool/assets/68882142/80359e61-f0ba-4349-8f1a-84614971b7c2)
